test/modules/aws-elasticache-integration.ts

```diff
@@ -114,24 +119,28 @@ describe('Elasticache Integration Testing', () => {
     ),
   );
 
-  it(
-    'tries to update cache_cluster node type',
-    query(`
-  UPDATE cache_cluster SET node_type='cache.t2.small' WHERE cluster_id='${clusterId}'
-  `),
-  );
+  it('tries to update cache_cluster node type', done => {
+    query(`  
+    UPDATE cache_cluster SET node_type='${updatedNodeType}' WHERE cluster_id='${clusterId}';
+    `)((e?: any) => {
+      if (!!e) return done(e);
+      done();
+    });
+  });
 
   it('applies the cache_cluster node_type update', apply());
 
-  it(
-    'checks that cache_cluster have been modified',
+  it('checks that cache_cluster have been modified', done => {
     query(
       `
-  SELECT * FROM cache_cluster WHERE node_type='cache.t2.small';
+  SELECT * FROM cache_cluster WHERE cluster_id='${clusterId}' AND node_type='${updatedNodeType}';
 `,
       (res: any) => expect(res.length).toBe(1),
```

Comment:
```suggestion
      (res: any) => expect(res.length).toBe(1)
```

Can you delete this trailing comma here? I suspect JS shenanigans.

Reply:
Nevermind on this, I am wrong.